support/fixtures/factories/contxtRole.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+
+factory
+  .define('contxtRole')
+  .option('fromServer', false)
+  .sequence('id')
```

Comment:
Based on the typedef, I think this would be a UUID created in the `attrs` section